src/client/public-openfin/rt-fx.json

```diff
@@ -13,7 +13,8 @@
     }
   },
   "runtime": {
-    "version": "17.85.53.10"
+    "version": "17.85.53.10",
```

Comment:
how about changing the uuid/name ... I think that might be a more straightforward way of achieving distinct contexts .. security realm is usually about segregating runtimes / IABus traffic (I think .. it's been a while 😄)

Reply:
I will test interop

Reply:
good call! interop is in fact broken with the differing security realm. I'll work on an alternate fix.